packages/web/src/fixtures/utility/string.ts

```diff
@@ -0,0 +1 @@
+export const truncate = (str: string, len: number) => (str.length <= len ? str : str.substr(0, len) + '...')
```

Comment:
This function is placeholder in loading?

Reply:
I implemented this function to match Figma's design.